src/examples/RetryAddPointExample.java

```diff
@@ -0,0 +1,82 @@
+package net.opentsdb.examples;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+import net.opentsdb.core.TSDB;
+import net.opentsdb.core.WritableDataPoints;
+import net.opentsdb.utils.Config;
+
+import com.stumbleupon.async.Deferred;
+
+public class RetryAddPointExample {
+	private static String pathToConfigFile;
+
+	public static void processArgs(String[] args) {
+		/*
+		 * Set these as arguments so you don't have to keep path information in source files
+		 */
+		if (args.length == 0) {
+			System.err
+					.println("First (and only) argument must be the full path to the opentsdb.conf file. (e.g. /User/thisUser/opentsdb/src/opentsdb.conf");
```

Comment:
Actually it doesn't have to be as the config can load from some default locations. This should be optional along with a Usage command.